### PR TITLE
feat(templates): Added profile field to templates and displaying prof…

### PR DESCRIPTION
**Pull request by eozan**
…ile in list and authoring.

SDPA-8

### CODE REVIEW

**Comment by mugurrus** [comment]
:+1:

File: client/app/scripts/superdesk-archive/views/media-box-list.html [hunk 1]
```diff
@@ -27,6 +27,7 @@
             <time sd-datetime data-date="item.versioncreated"></time>
         </div>
         <div class="line">
+            <div class="label label--{{item.profile}}" ng-if="item.profile">{{ item.profile }}</div>
```
Comment (petrjasek) [inline client/app/scripts/superdesk-archive/views/media-box-list.html]: just wondering - why double dash in class name?

Comment (eozan) [inline client/app/scripts/superdesk-archive/views/media-box-list.html]: It's what Vladimir told me to use, might be a convention maybe.

Comment (petrjasek) [inline client/app/scripts/superdesk-archive/views/media-box-list.html]: ok

**Pull request merged**